src/driver.rs

```diff
@@ -163,33 +166,7 @@ impl rustc_driver::Callbacks for ClippyCallbacks {
 }
 
 fn display_help() {
-    println!(
-        "\
-Checks a package to catch common mistakes and improve your Rust code.
-
-Usage:
-    cargo clippy [options] [--] [<opts>...]
-
-Common options:
-    -h, --help               Print this message
-        --rustc              Pass all args to rustc
```

Comment:
There is also this flag, which also forwards `--help` and `--version` to `rustc`.